### PR TITLE
added container instrumentation documentation

### PR DESCRIPTION
**Pull request by vikramraman**

### CODE REVIEW

File: docs/container.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Container Instrumentation
+
+This document details the steps required to instrument a Java based container to use the opentracing [specialagent](https://github.com/opentracing-contrib/java-specialagent).
```
Comment (rkempf42) [inline docs/container.md]: You can instrument a Java-based container to use the [OpenTracing Special Agent](https://github.com/opentracing-contrib/java-specialagent) by following the steps on this page. 
OR:
To instrument.... follow the steps...

File: docs/container.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+## Introduction
```
Comment (rkempf42) [inline docs/container.md]: Is this really the best heading? 
Maybe instead ## About the Dockerfile

File: docs/container.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+The steps below explain how to modify the Dockerfile for a Java application to automatically instrument your application using the opentracing specialagent.
```
Comment (rkempf42) [inline docs/container.md]: This page explains how to modify the Dockerfile (?why not Docker file?) for a Java application to instrument your application using the OpenTracing Special Agent. 
??What's the correct capitalization for this? It feels like a name. And, OpenTracing, not opentracing for sure.

File: docs/container.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+```
+#!/bin/bash
+java -javaagent:/app/opentracing-specialagent-1.5.1.jar \
+    -Dsa.tracer=wavefront \
+    -Dwf.application=myApplication \
+    -Dwf.service=myService \
+    -Dwf.reportingMechanism=direct
+    -Dwf.server=https://YOUR_CLUSTER.wavefront.com
+    -Dwf.token=<YOUR_WAVEFRONT_TOKEN>
```
Comment (hanwavefront) [inline docs/container.md]: `\` at the end of lines 60-62

File: docs/container.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+Edit the `run.sh` file as follows:
+
+```
+#!/bin/bash
+java -javaagent:/app/opentracing-specialagent-1.5.1.jar \
+    -Dsa.tracer=wavefront \
+    -Dwf.application=myApplication \
+    -Dwf.service=myService \
+    -Dwf.reportingMechanism=direct
+    -Dwf.server=https://YOUR_CLUSTER.wavefront.com
```
Comment (hanwavefront) [inline docs/container.md]: nit: `https://<YOUR_CLUSTER>.wavefront.com` might be slightly preferable to `https://YOUR_CLUSTER.wavefront.com`

File: docs/container.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+RUN apt-get -y update
+RUN apt-get install -y openjdk-8-jdk
+
+COPY target/my-java-application.jar /app/my-java-application.jar
+
+# Install wget
+RUN apt-get-install -y wget
+
+# Download the latest stable release of the special agent as per https://github.com/opentracing-contrib/java-specialagent
```
Comment (hanwavefront) [inline docs/container.md]: nit: capitalize `special agent` to be `Special Agent`

**Pull request merged**